src/app/give-help/material-aid-form/material-aid-form.component.ts

```diff
@@ -24,13 +24,19 @@ export class MaterialAidFormComponent {
   phonePrefix: string = '48';
   phoneNumber: string = '';
   loading: boolean = false;
+  @ViewChild('phoneInput')
+  phoneInput!: { nativeElement: { value: any } };
```

Comment:
Poprawnie otypuj `nativeElement`.

Reply:
@mgol  Dałem typ ElementRef i jest ok z tym HTMLInputElement niezbyt to działało, nie chcę już przy tym dłubać tym bardziej że nie tego dotyczy ticket a to byl jedynie improvement przy okazji. Jeżeli podpinamy tu lib to i tak to jest temporary.